### DISCUSSION TITLE
Add CTRL+Y Shortcut to "Create New Tab Next to Current"

### DISCUSSION THREAD

**Discussion by ekk88d** [fwextensions/QuicKey#61] [Ideas]
Hi,

Since there isn't yet discussion going here, I'm going to use it to suggest a QuicKey feature:

Add CTRL+Y functionality to "Create New Tab Next to Current." There are separate Chrome extensions that enable just this one function, so it's a perfect addition to QuicKey.

Cheers and Thanks

**Comment by fwextensions** [comment]
Sorry for the slow response.

QuicKey is mostly about finding and switching to/opening existing tabs, bookmarks and history items, rather than creating new ones.  So this feels a bit like feature creep, though it would be easy to add.  If it was added, however, it wouldn't have a default <kbd>ctrl-Y</kbd> shortcut, as apparently extensions are limited to 4 suggested shortcuts, and QuicKey already has that many.  So you'd have to manually set it to something after installing the extension, which would make it somewhat less convenient.
